From b4c4c686e048c783df56b3c9e4ac16cebfbc7658 Mon Sep 17 00:00:00 2001
From: Kaspar Winckler <155115050+KasparWinckler@users.noreply.github.com>
Date: Sun, 14 Sep 2025 16:29:40 +0200
Subject: [PATCH] adapt make-linux.mk to LibreELEC

---
 make-linux.mk | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/make-linux.mk b/make-linux.mk
index 5ac8a7d95..8ac881839 100644
--- a/make-linux.mk
+++ b/make-linux.mk
@@ -9,7 +9,7 @@ ifeq ($(origin CXX),default)
 	CXX:=$(shell if [ -e /opt/rh/devtoolset-8/root/usr/bin/g++ ]; then echo /opt/rh/devtoolset-8/root/usr/bin/g++; else echo $(CXX); fi)
 endif
 
-INCLUDES?=-Irustybits/target -isystem ext -Iext/prometheus-cpp-lite-1.0/core/include -Iext-prometheus-cpp-lite-1.0/3rdparty/http-client-lite/include -Iext/prometheus-cpp-lite-1.0/simpleapi/include
+INCLUDES+=-Irustybits/target -isystem ext -Iext/prometheus-cpp-lite-1.0/core/include -Iext-prometheus-cpp-lite-1.0/3rdparty/http-client-lite/include -Iext/prometheus-cpp-lite-1.0/simpleapi/include
 DEFS?=
 LDLIBS?=
 DESTDIR?=
@@ -87,7 +87,7 @@ else
 	CXXFLAGS?=-O3 -fstack-protector
 	override CXXFLAGS+=-Wall -Wno-deprecated -std=c++17 -pthread $(INCLUDES) -DNDEBUG $(DEFS)
 	LDFLAGS?=-pie -Wl,-z,relro,-z,now
-	ZT_CARGO_FLAGS=--release
+	ZT_CARGO_FLAGS+=--release
 endif
 
 ifeq ($(ZT_QNAP), 1)
@@ -122,7 +122,7 @@ ifeq ($(ZT_VAULT_SUPPORT),1)
 endif
 
 # Determine system build architecture from compiler target. This is hairy due to "ARM wrestling."
-CC_MACH=$(shell $(CC) -dumpmachine | cut -d '-' -f 1)
+CC_MACH=${ARCH}
 ZT_ARCHITECTURE=999
 ifeq ($(CC_MACH),x86_64)
 	ZT_ARCHITECTURE=2
@@ -313,7 +313,7 @@ ifeq ($(ZT_SSO_SUPPORTED), 1)
 		ifeq ($(ZT_DEBUG),1)
 			LDLIBS+=rustybits/target/debug/libzeroidc.a -ldl -lssl -lcrypto
 		else
-			LDLIBS+=rustybits/target/release/libzeroidc.a -ldl -lssl -lcrypto
+			LDLIBS+=.${TARGET_NAME}/target/${TARGET_NAME}/release/libzeroidc.a -ldl -lssl -lcrypto
 		endif
 	endif
 endif
-- 
2.43.0

