From 01035b047abbd1f5e5821b9035fcfb5fa19bbff1 Mon Sep 17 00:00:00 2001
From: Kaspar Winckler <155115050+KasparWinckler@users.noreply.github.com>
Date: Mon, 15 Jan 2024 21:10:05 +0100
Subject: [PATCH] LibreELEC notification

---
 daemon/notify.go | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/daemon/notify.go b/daemon/notify.go
index e57100c..639c4f6 100644
--- a/daemon/notify.go
+++ b/daemon/notify.go
@@ -52,6 +52,15 @@ func notify(id int64, body string) error {
 		cmd = exec.Command(
 			"kdialog", "--title", summary, "--passivepopup", body, "--icon", IconPath, "3",
 		)
+	} else if internal.IsCommandAvailable("kodi-send") {
+		cmd = exec.Command(
+			"kodi-send", fmt.Sprintf("--action=RunScript(service.nordvpn, notification, %s, %s, %s)", summary, body, IconPath),
+		)
+		out, err := cmd.CombinedOutput()
+		if err != nil {
+			return errors.New(strings.Trim(string(out), "\n"))
+		}
+		return nil
 	} else {
 		return nil
 	}
-- 
2.34.1

