From e29751c3080f29767d50bdc5a0d24e1ed301a8bf Mon Sep 17 00:00:00 2001
From: Kaspar Winckler <155115050+KasparWinckler@users.noreply.github.com>
Date: Fri, 29 Mar 2024 10:48:14 +0100
Subject: [PATCH 1/2] LibreELEC constants

---
 internal/constants.go | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/internal/constants.go b/internal/constants.go
index a55280f..9e99240 100644
--- a/internal/constants.go
+++ b/internal/constants.go
@@ -26,7 +26,7 @@ const (
 	TmpDir = "/tmp/"
 
 	// NordvpnGroup that can access daemon socket
-	NordvpnGroup = "nordvpn"
+	NordvpnGroup = "root"
 
 	// PermUserRWX user permission type to read write and execute
 	PermUserRWX = 0700
@@ -95,18 +95,18 @@ var (
 	RunDir = PrefixCommonPath("/run/nordvpn")
 
 	// LogPath defines where logs are located if systemd isn't used
-	LogPath = PrefixDataPath("/var/log/nordvpn")
+	LogPath = PrefixDataPath("/run/log/nordvpn")
 
 	// AppDataPath defines path where app data is stored
-	AppDataPath = PrefixDataPath("/var/lib/nordvpn")
+	AppDataPath = PrefixDataPath("/storage/.kodi/userdata/addon_data/service.nordvpn")
 
 	// AppDataPathCommon defines path where common app data files are stored. These files may
 	// be removed after every app update
-	AppDataPathCommon = PrefixCommonPath("/var/lib/nordvpn")
+	AppDataPathCommon = PrefixCommonPath("/storage/.kodi/userdata/addon_data/service.nordvpn")
 
 	// AppDataPathStatic defines path where static app data (such as helper executables) are
 	// stored. Normally it is the same as AppDataPath
-	AppDataPathStatic = PrefixStaticPath("/var/lib/nordvpn")
+	AppDataPathStatic = PrefixStaticPath("/storage/.kodi/userdata/addon_data/service.nordvpn")
 
 	UsrBinPathStatic = PrefixStaticPath("/usr/bin")
 
-- 
2.34.1

